Server/server.js

```diff
@@ -58,10 +55,45 @@ app.get('/:id', (req, res) => {
 app.get('/api/reviews/:id', (req, res) => {
   ReviewsController.getById(req.params.id, (err, result) => {
     if (err) {
-      console.log('ERROR')
+      console.log('ERROR');
       throw new Error();
     } else {
       res.send(result);
     }
   });
 });
+
+app.post('/api/reviews', (req, res) => {
+  ReviewsController.createReview(req.body, (err, result) => {
+    if (err) {
+      console.log('ERROR');
+      throw new Error();
+    } else {
+      res.send(result);
+    }
+  })
+});
+
+app.put('/api/reviews/:id', (req, res) => {
+  ReviewsController.updateById(req.params.id, req.body, (err, result) => {
+    if (err) {
+      res.send(err);
```

Comment:
I don't actually know the proper way to handle errors but I think you may want to be consistent across all your endpoints, one way or another.